src/features/pastelStatistics/components/chart/EChartsLineChart.tsx

```diff
@@ -164,6 +166,9 @@ export const EChartsLineChart = (props: TLineChartProps): JSX.Element => {
   return (
     <div className={styles.container}>
       <div className={styles.lineChartHeader}>
+        <Link to={routes.STATISTICS} className={styles.backButtonLink}>
+          <span className={styles.arrowBackButton}></span>
+        </Link>
```

Comment:
can we use the fontawesome icon for the back button? Because we already added/used `"@fortawesome/fontawesome-free": "5.12.0"` in the wallet. (see the sidebar) https://fontawesome.com/v5.15/icons/arrow-circle-left?style=solid

And I think we can create a component and re-use it instead of adding the same code for every chart.